openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -60,35 +60,38 @@ This specification uses the terms "Access Token", "Authorization Endpoint", "Aut
 
 This specification also defines the following terms. In the case where a term has a definition that differs, the definition below is authoritative for this specification.
 
-Credential:
-:  A set of one or more claims about a subject made by a Credential Issuer. Note that this definition of a term "credential" in this specification is different from that in [@!OpenID.Core] and [@!RFC6749].
+Credential Dataset:
+:  A collection of one or more claims about a subject, provided by a Credential Issuer.
 
-Verifiable Credential (VC):
-:  An Issuer-signed Credential whose integrity can be cryptographically verified. It can be of any format used in the Issuer-Holder-Verifier Model, including, but not limited to those defined in [@VC_DATA] and [@ISO.18013-5].
+Credential (or Verifiable Credential):
+:  A Credential Dataset signed by an Issuer, whose integrity can be cryptographically verified. This can be in any Credential Format. In this specification, the term "Verifiable Credential" is also referred to as "Credential". Note that this definition of the term "Credential" is different from that in [@!OpenID.Core] and [@!RFC6749]. In this specification, Credential is not used in any other meaning, e.g., passwords as login credentials.
 
-W3C Verifiable Credential:
-:  A Verifiable Credential compliant to the [@VC_DATA] specification.
+Credential Format:
+:  A format used to represent a Verifiable Credential. The exact parameters required to describe a Credential Format are defined in the Credential Format Profile. Definition of Credential Formats is out of scope for this specification. Examples for Credential Formats are W3C Verifiable Credentials [@VC_DATA], SD-JWT VC [@!I-D.ietf-oauth-sd-jwt-vc] and ISO/IEC 18013-5 mDL [@ISO.18013-5].
 
-Presentation:
-: Data that is presented to a specific verifier, derived from one or more Verifiable Credentials that can be from the same or different Credential Issuers.
+Credential Format Profile:
+:  Sets of parameters specific to individual Credential Formats. Credential Format Profiles for W3C Verifiable Credentials [@VC_DATA] and ISO/IEC 18013-5 mDL [@ISO.18013-5] are provided in (#format-profiles). Other specifications or deployments can define their own Credential Format Profiles using the extension points defined in this specification.
+
+Credential Format Identifier:
+:  An identifier for a specific Credential Format in the context of this specification. Implies the usage of the respective Credential Format Profile specific parameters.
 
-Verifiable Presentation (VP):
-:  A Holder-signed Credential whose integrity can be cryptographically verified to provide Cryptographic Holder Binding. It can be of any format used in the Issuer-Holder-Verifier Model, including, but not limited to those defined in [@VC_DATA] and [@ISO.18013-5].
+Credential Configuration:
```

Comment:
Just to test the thinking here, are we for seeing the situation where a single credential issuer would be able to issue more then one credential dataset from a single credential configuration. For example if we consider a delegation scenario of birth certificates where as a holder I want to be able to request my birth certificate but im also authorised to request my son's, under this terminology these two credentials would have different datasets but could share a credential configuration?

Reply:
Agreed. Even easier, you have two cars and therefore two vehicle registration VCs. They have the same Credential Configuration but different Credential Dataset

Reply:
Reviewing the discussions in Rome, I think there is broad consensus on this